tests/unit/config/config_test.py

```diff
@@ -349,6 +350,18 @@ def test_config_invalid_environment_dict_key_raises_validation_error(self):
                 )
             )
 
+    def test_load_yaml_with_yaml_error(self):
+        tmpdir = py.test.ensuretemp('invalid_yaml_test')
+        invalid_yaml_file = tmpdir.join('docker-compose.yml')
+        invalid_yaml_file.write("""
```

Comment:
The test is a lot more clear when you can see the contents of the file inline with the test. You don't have to go digging around for the file to see what it's testing.

It also prevents someone else from incorrectly using this file for another test case.

An invalid `yaml` file would also cause the `pre-commit` check to fail. We could go adding exceptions for these, but it's unnecessary if we inline it.

I can see the advantage to a separate file if the file is large enough that editing it inline is annoying. For something this small I don't see any advantage to having a separate file.